.azure-pipelines/azure-pipelines.yml

```diff
@@ -0,0 +1,44 @@
+# Node.js
```

Comment:
Should this be renamed as well? Perhaps mgmt?

Reply:
by the way this file move is what is breaking the current PR validation build for this definition.